iOS/DZNEmptyDataSet/source/DZNEmptyDataSet/DZNEmptyDataSet.csproj

```diff
@@ -46,15 +46,13 @@
     <ObjcBindingCoreSource Include="StructsAndEnums.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ObjcBindingNativeLibrary Include="..\..\externals\libDZNEmptyDataSet.a">
-      <Link>libDZNEmptyDataSet.a</Link>
-    </ObjcBindingNativeLibrary>
+    <Compile Include="Additions.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="libDZNEmptyDataSet.linkwith.cs">
-      <DependentUpon>libDZNEmptyDataSet.a</DependentUpon>
-    </Compile>
-    <Compile Include="Additions.cs" />
+    <NativeReference Include="DZNEmptyDataSet.framework">
```

Comment:
I don't think these changes are correct when using the .a file from the pod

Reply:
ok, I'll fix that.